lib/src/cnf/mod.rs

```diff
@@ -35,3 +35,9 @@ pub const SERVER_NAME: &str = "SurrealDB";
 
 /// Datastore processor batch size for scan operations
 pub const PROCESSOR_BATCH_SIZE: u32 = 50;
+
+/// Forward all signup/signin query errors to a client trying authenticate to a scope. Do not use in production.
+pub static INSECURE_FORWARD_SCOPE_ERRORS: Lazy<bool> = Lazy::new(|| {
+	let val = std::env::var("INSECURE_FORWARD_SCOPE_ERRORS").unwrap_or("false".to_string());
```

Comment:
let's scope it with `SURREAL_` like the rest of env vars

```suggestion
	let val = std::env::var("SURREAL_INSECURE_FORWARD_SCOPE_ERRORS").unwrap_or("false".to_string());
```

Reply:
Yep, good one!